src/main.js

```diff
@@ -175,11 +174,11 @@ import AppMain from './components/AppMain'
     Vue.prototype.t = window.t
 
     // Start the app once document is done loading
-    $(document).ready(function () {
+    document.addEventListener("DOMContentLoaded", function(event) {
```

Comment:
I am unsure if this really needed. It will most probably not cause issues but I found nowhere the requirement documented.

Reply:
Me neither. We could remove this and try if it still works

Reply:
Let's see what comes out from [my question](https://forum.vuejs.org/t/waiting-for-dom-ready/109120). I fear it will be more of unstable behavior and less of no function at all.

Reply:
That’s what I would expect, too. Good idea to ask in the vue forum!

Reply:
I already asked in the Discourse app but there was only an uncertain statement given.

Reply:
funny how nobody seems to know